src/unitxt/artifact.py

```diff
@@ -432,17 +433,18 @@ def __str__(self):
 def fetch_artifact(artifact_rep) -> Tuple[Artifact, Union[Artifactory, None]]:
     if isinstance(artifact_rep, Artifact):
         return artifact_rep, None
-    if Artifact.is_artifact_file(artifact_rep):
+    if isinstance(artifact_rep, str) and Artifact.is_artifact_file(artifact_rep):
```

Comment:
IIUC, this code is used to load an artifact from string name "tasks.classification.multi_class" and the code below is if it's "tasks.classification.multi_class[metrics=[....]].  Can remove this line and let the code below handle both cases?